data/src/main/java/utill/DMAPIhandler.kt

```
@@ -0,0 +1,66 @@
+                else -> OtherHttpException(
+                    message = message,
+                    code = e.code()
+                )
+            }
+        } catch (e: SocketTimeoutException) { // 408 : 요청시간초과
+            throw TimeOutException(message = e.message)
+        } catch (e: UnknownHostException) { // 클라이언트 인터넷 없음 에러
+            throw NoInternetException()
+        } catch (e: NeedLoginException) { // 토큰 만료
+            throw NeedLoginException()
+        } catch (e: Exception) {  // 알려지지 않은 에러
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)